rules/palettes.yaml

```diff
@@ -20,6 +20,30 @@
 	PaletteFromCurrentTileset@decoration:
 		Name: terraindecoration
 		ShadowIndex: 1
+	PaletteFromFile@apolblue:
+		Name: apolblue
+		Filename: apolblue.pal
+		Alpha: 0.8
```

Comment:
Will check coming week or sunday, did not noticed this wont work. maybe not needed for that anyway. So rather removing the Alpha value.